src/components/SearchInput/index.js

```diff
@@ -27,11 +40,15 @@ class SearchInputContainer extends React.Component {
     clearTimeout(this.timer);
   }
 
+  props: Props;
```

Comment:
React.Component<Props, State> tells the component class it use Props type for this.props, therefore don't need this flow type declaration any more.